src/main/java/com/google/firebase/messaging/LightSettings.java

```diff
@@ -27,7 +27,7 @@
 public final class LightSettings {
   
   @Key("color")
-  private final Color color;
+  private final LightSettingsColor lightSettingsColor;
```

Comment:
Lets call the variable name `color`. And also change the setters back to `setColor` and `setColorFromString`.